GageStatsAgent/GageStatsAgent.cs

```diff
@@ -208,13 +208,26 @@ public IQueryable<String> GetRoles()
 
         #endregion 
         #region Station
-        public IQueryable<Station> GetStations()
+        public IQueryable<Station> GetStations(List<string> stationTypeList = null, List<string> agencyList = null, int page = 1)
         {
-            return Select<Station>().Include(s=>s.StationType).Include(s=>s.Agency);
+            // set number of items to skip/return for pagination
+            var numItemsReturned = 50;
+            var skip = (page - 1) * numItemsReturned;
+
+            var query = this.Select<Station>();
+            // if filters, apply them before returning query
+            if (stationTypeList.Any() == true)
```

Comment:
`StationTypeList.Any()` is a boolean, so its duplicative in writing `if (true==true)` of `(false == true)`, would recommend changing it to `if (stationTypeList.Any())` or if you are checking if false, then you can use the ! character such as `if (!stationTypeList.Any())`